src/GovUk.Education.ExploreEducationStatistics.Public.Data.Api.Tests/Services/ContentApiClientTests.cs

```diff
@@ -0,0 +1,130 @@
+        _logger = new Mock<ILogger<ContentApiClient>>();
+        _contentApiClient = new ContentApiClient(_logger.Object, _client);
+    }
+
+    [Fact]
+    public async Task ListPublications_HttpClientRespondsWithBadRequest_ReturnsBadRequestObjectResultAndLogsError()
+    {
+        _mockHttp.Expect(HttpMethod.Post, "http://localhost/api/publications")
+            .Respond(HttpStatusCode.BadRequest, new StringContent("test message"));
+
+        var response = await _contentApiClient.ListPublications(
+            It.IsAny<int>(), 
+            It.IsAny<int>(), 
+            It.IsAny<string>(),
+            It.IsAny<IEnumerable<Guid>>());
+
+        _mockHttp.VerifyNoOutstandingExpectation();
+
+        _logger.Verify(logger => logger.Log(
+                It.Is<LogLevel>(logLevel => logLevel == LogLevel.Error),
+                It.Is<EventId>(eventId => eventId.Id == 0),
+                It.Is<It.IsAnyType>((@object, @type) =>
+                    @object.ToString() == 
+                        $"""
+                         Failed to retrieve publications with status code: {HttpStatusCode.BadRequest}. Message:
+                         test message
+                         """.TrimIndent()),
+                It.IsAny<Exception>(),
+                It.IsAny<Func<It.IsAnyType, Exception?, string>>()),
+            Times.Once);
+
+        var left = response.AssertLeft();
+        Assert.IsType<BadRequestObjectResult>(left);
+    }
+
+    [Theory]
+    [InlineData(HttpStatusCode.RequestTimeout)]
+    public async Task
+        ListPublications_HttpClientRespondsWithUnsuccessfulStatusCode_ThrowsHttpRequestExceptionAndLogsError(
+            HttpStatusCode responseStatusCode)
+    {
+        _mockHttp.Expect(HttpMethod.Post, "http://localhost/api/publications")
+            .Respond(responseStatusCode, new StringContent("test message"));
+
+        var action = async () => await _contentApiClient.ListPublications(
+            It.IsAny<int>(),
+            It.IsAny<int>(),
+            It.IsAny<string>(), 
+            It.IsAny<IEnumerable<Guid>>());
+
+        await Assert.ThrowsAsync<HttpRequestException>(action);
+
+        _mockHttp.VerifyNoOutstandingExpectation();
+
+        _logger.Verify(logger => logger.Log(
+                It.Is<LogLevel>(logLevel => logLevel == LogLevel.Error),
+                It.Is<EventId>(eventId => eventId.Id == 0),
+                It.Is<It.IsAnyType>((@object, @type) =>
+                    @object.ToString() == 
+                        $"""
+                         Failed to retrieve publications with status code: {responseStatusCode}. Message:
+                         test message
+                         """.TrimIndent()), It.IsAny<Exception>(),
+                It.IsAny<Func<It.IsAnyType, Exception?, string>>()),
+            Times.Once);
+    }
+
+    [Fact]
+    public async Task ListPublications_HttpClientRespondsWithSuccess_ReturnsPublications()
```

Comment:
Similar to above, could simplify name to `ListPublications_HttpClientSuccess_ReturnsPublications`